sbtStudentCommands/Navigation.scala.template

```diff
@@ -104,6 +104,25 @@ object Navigation {
 
   private lazy val ExerciseNr = OptSpace ~> IntBasic.?
 
+  def gotoFirstExercise: Command = Command.command("gotoFirstExercise") { state =>
+    val firstExercise = state.get(mapNext).get.keys.toVector.sorted.head
+    withZipFile(state, firstExercise) { () =>
+      val src = new sbt.File(new sbt.File(Project.extract(state).structure.root), "exercises")
```

Comment:
Guessing this is basically a copy/paste from gotoExerciseNr. Goto First Exercises should probably just figure out the exercise number and then call gotoExerciseNr under the hood to eliminate duplication.